src/explore-education-statistics-frontend/src/pages/_error.tsx

```diff
@@ -82,24 +82,7 @@ class ErrorPage extends Component<Props> {
   }
 
   public render() {
-    const { errorMessage, statusCode } = this.props;
-    const pageTitle = errorMessage
-      ? ErrorPage.statusCodeTitles[500]
-      : ErrorPage.statusCodeTitles[statusCode];
-
-    return (
-      <Page>
-        {errorMessage ? (
-          <>
-            <PageTitle title={pageTitle} />
-
-            <p>{errorMessage}</p>
-          </>
-        ) : (
-          this.getStatusCodePage()
-        )}
-      </Page>
-    );
+    return <Page>{this.getStatusCodePage()}</Page>;
```

Comment:
Would just move all of the `getStatusCodePage` into the `render` method

Reply:
Yep it might as well think of it, done.